app/src/main/java/fr/free/nrw/commons/media/MediaDetailPagerFragment.java

```diff
@@ -173,11 +185,33 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 // Set wallpaper
                 setWallpaper(m);
                 return true;
+            case R.id.menu_change_color_background:
+                //Toast.makeText(getActivity(), getActivity().toString(), Toast.LENGTH_SHORT).show();
+                changeBackGroundColor();
+                return true;
             default:
                 return super.onOptionsItemSelected(item);
         }
     }
 
+    /**
+     * Set the background color to white
+     */
+    private void changeBackGroundColor(){
+
+
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getActivity());
+        Boolean theme = prefs.getBoolean("theme", true);
```

Comment:
@domdomegg This will not work anymore due to the change in #3460

Reply:
@macgills @domdomegg Maybe we can close this PR!

Reply:
It would be nice if this pr could be completed. The issue still persists I would say

Reply:
@macgills Seems like the author is not available at this time!

Reply:
https://github.com/commons-app/apps-android-commons/issues/3583 I have created this to help ameliorate the situation.

@madhurgupta10 do you want to work on this issue or is this just a desire to see stale PRs closed?

Reply:
@macgills I can work on the issue but I think the author or this PR already did a lot of work on this, I would be happy if we can complete this PR either the author himself or someone else makes the changes. It's better if we close it as soon as possible otherwise the complexity will increase.

Reply:
If you want to work on this I can assign you. If you were to work on it good practice would be to use the fork of the author as your starting point, most likely. 

That said it is 42 lines. Probably easier to redo.

The author seems unavailable but I guess this PR will still show up in the ticket. Comment on the ticket if you want to take it. I'll close this PR